stack_grid:
  global_frame: map
  robot_base_frame: base_link
  resolution: 0.15
  height: 50
  width: 50

  update_frequency: 10

  track_unknown: true
  threshold_occupancy: 50
  stack_policy: -1
  inflation_enable: false
  inflation_radius: 1
  inscribed_radius: 0.5
  static_sources: input1 input2 input3 input4 input5 fake
  input1: {msg_type: OccupancyGrid, topic: /map/image_segmentation}
  input2: {msg_type: OccupancyGrid, topic: /map/scan}
  input3: {msg_type: OccupancyGrid, topic: /map/rear_road_curb}
  input4: {msg_type: OccupancyGrid, topic: /map/front_road_curb}
  input5: {msg_type: OccupancyGrid, topic: /map/realsense_lidar}
  # fake: {msg_type: OccupancyGrid, topic: /map/abc}